### PR TITLE
Fix grid

### PR DESCRIPTION
**Pull request by softguru555**
Change the way of displaying  the answers to grid one

### CODE REVIEW

File: frontend/eslint.config.mjs
Comment (snowgy920) [inline frontend/eslint.config.mjs:1]: Why is this file included in the commit?

Comment (softguru555) [reply frontend/eslint.config.mjs:1]: It was included by mistake. it's removed and pushed again.

File: frontend/app/main/discuss/page.tsx [hunk 1]
```diff
@@ -59,36 +59,18 @@ const Discuss: React.FC = () => {
         </div>
       </div>
 
-      <div className="p-4 sm:p-6 xl:p-12.5">
-        <div className="grid grid-cols-1 gap-y-6 gap-x-4 md:gap-x-6 xl:grid-cols-2 xl:gap-x-7.5">
+      <div className="p-4 sm:p-6 w-full">
+        <div className=" gap-y-6 gap-x-4 md:gap-x-6  xl:gap-x-7.5">
```
Comment (snowgy920) [inline frontend/app/main/discuss/page.tsx:63]: there is a space before gap-y-6

**Review by snowgy920 — COMMENTED** [review]
problem still persist.

image

**Review by snowgy920 — CHANGES_REQUESTED** [review]
Can you keep the consistency of the button style? make it one-line. 

image

File: frontend/components/AccordionItemOne.tsx [hunk 2]
```diff
@@ -162,12 +162,13 @@ const AccordionItemOne: React.FC<FaqItem> = ({ active, handleToggle, faq }) => {
...
-            <div className='flex row-span-1'>
-
-              <button className="flex w-25 items-center gap-2 rounded-sm py-1.5 px-4 text-left text-sm hover:bg-gray dark:hover:bg-meta-4"
+          <div key={key} className='gap-4 border-b'>
+            <div className='grid grid-rows-1 grid-cols-5'>
+              <div className='row-span-2 col-span-1'>{element.email || ""}:</div>
+              <div className="text-lg row-span-1 col-span-4" >{element.answer || ""}</div>
+            </div>
+            <div className="flex flex-row-reverse">
+              <button className="w-25 items-center gap-2 rounded-sm py-1.5 px-4 text-left text-sm hover:bg-gray dark:hover:bg-meta-4"
```
Comment (snowgy920) [inline frontend/components/AccordionItemOne.tsx:171]: why did you remove `flex` class from here? if you keep it, it will be in one line. any specific reason of removing `flex`?

File: frontend/components/AccordionItemOne.tsx
Comment (snowgy920) [inline frontend/components/AccordionItemOne.tsx:1]: looks weird.

**Review by snowgy920 — APPROVED** [review]
